### PR TITLE
Fixes borg flashlight color picker not working

### PR DESCRIPTION
**Pull request by zxaber**
## About The Pull Request
Added two new procs, `flashlight_toggle()` and `set_flashlight_color()` to modular computers, to handle their related functions after being called from `ui_act()`. Borg tablets now override these procs instead of `ui_act()`, which fixes borg not being able to change their lamp color (or use the tablet to toggle their lamp).

~~Removes the parent call for borg tablets ui_act() that was added in with the rest of #53964 and led to breaking the borg flashlight color picker (and tablet light button, though that one is less important). The borg tablet doesn't need this parent call to prevent ghosts from using the tablet because the tablet is not accessible to any ghost.~~

## Why It's Good For The Game
Fix bug.

## Changelog
:cl:
fix: Borgs can choose their flashlight colors again.
/:cl:

### CODE REVIEW

**Comment by TheBonded** [comment]
Borgs could choose flashlight colours?

**Comment by zxaber** [comment]
As of the recent PR that added borg tablets, yes. It lasted all of about three days before someone broke it.

File: code/modules/modular_computers/computers/item/tablet.dm [hunk 1]
```diff
@@ -114,12 +114,10 @@
 	.["light_on"] = borgo?.lamp_enabled
 	.["comp_light_color"] = borgo?.lamp_color
 
-//Overrides the ui_act to make the flashlight controls link to the borg instead
+//Overrides the ui_act to make the flashlight controls link to the borg instead.
+//No need to call parent, as this device is made inaccessable from anyone that shouldn't
+//be using it via other means (inaccessable to ghosts, shuts down while borg is offline, etc).
```
Comment (Rohesie) [inline code/modules/modular_computers/computers/item/tablet.dm:119]: Can I still use it if I'm playing on the borg, have the window open and ghost/aghost?

Comment (zxaber) [reply code/modules/modular_computers/computers/item/tablet.dm:119]: Nope, the window goes out of range (eye turns orange) as soon as you become a ghost.

Comment (Rohesie) [reply code/modules/modular_computers/computers/item/tablet.dm:119]: That's nice. It's still not ideal to redefine the proc without calling the parent.
Can you tweak the check that breaks this at the parent level, making a redefined proc there? That would make this code less brittle.

Comment (zxaber) [reply code/modules/modular_computers/computers/item/tablet.dm]: Flashlight setting on modular computers are now handled by calling the `toggle_flashlight()` and `set_flashlight_color()` procs. Borg tablets now override these and don't touch `ui_act()` at all.

**Pull request merged**